packages/cli/src/lib/commands/space.ts

```diff
@@ -115,6 +115,19 @@ export const space: CommandDefinition = (program) => {
 
             displayObject(await managerClient.disconnectHubs(opts), profileManager.getProfileConfig().format);
         });
+
+    spaceCmd
+        .command("delete")
```

Comment:
This should be in `si hub` command

Reply:
but I'm not sure because there is `si space disconnect <HUB_ID>`. Please consult with @a-tylenda

Reply:
I was also wondering whether it should be `space` or `hub`. I doubt that the user will look for deleting hub under the `space` command. I think `si hub delete <id>` would make more sense. But, as @patuwwy says, we do have `si space disconnect <hub-id>` on the other hand, we have `si sequence delete <seq-id>`... so, hard to say. 
I vote for `si hub delete <id>` ✋🏼 but I will not fight for it.

Reply:
maybe we should move both delete/disconnect to hub as this would make more sense? What do you think? @a-tylenda @patuwwy

Reply:
good idea 👍🏼 @patuwwy what do you think?

Reply:
Agree :)